app/views/content/early-years-teaching-training.md

```diff
@@ -19,89 +19,95 @@ date: "2021-03-29"
     arguments:
-      - Cost: "Self-funded"
+      - Cost: "This could be around £2,500 or £3,000 depending on the provider"
         Salary: "No"
-        Funding available: "No"
         Length of training: "3 months"
         Qualification: "Early years teacher status"
-      - "Graduate 'assessment only' training"
 ---
 
-As an early years teacher you'll work with young children up to the age of 5. Your input will have a lasting and positive effect on their development and wellbeing.
+The early years are critical in children’s development. As an early years teacher you’ll work with young children up to the age of 5. You’ll use your knowledge and skills to help young children enjoy high standards of teaching and open their minds to new ideas every day.
 
-The early years are critical in children's development. You’ll use your knowledge and skills to help young children enjoy high standards of teaching and open their minds to new ideas every day.
+You’ll need to do early years initial teacher training (EYITT) to achieve early years teacher status (EYTS). This will allow you to work in a nursery or early years setting. 
 
-## How to become an early years teacher
+If you want to teach in a primary school, including in a reception class, or secondary school then you will need to achieve qualified teacher status (QTS). [Find out more about training to teach primary and secondary pupils](/train-to-be-a-teacher).
 
-You’ll need to do 'early years initial teacher training' (EYITT) to achieve 'early years teacher status' (EYTS).
+## What you need to become an early years teacher
 
-EYTS is different from '[qualified teacher status](/what-is-qts)' (QTS), which is the equivalent for teaching children aged 3 to 18.
+There are 2 ways to train to become an early years teacher, either as a postgraduate if you already have a degree, or as part of an undergraduate degree. 
 
-Your training will enable you to meet the [early years teachers' standards](https://www.gov.uk/government/publications/early-years-teachers-standards).
+To do postgraduate training, you’ll need a bachelor’s degree or an equivalent, for example, a vocational, professional, or overseas qualification.
 
-To start early years teacher training, you’ll need these qualifications (or to demonstrate an equivalent standard):
+To start early years teacher training, you’ll need these GCSE qualifications (or a vocational, professional, or overseas qualification to the same standard):
 
 * maths grade 4 (C)
 * English grade 4 (C)
 * science grade 4 (C)
 
-If you do postgraduate early years teacher training, you'll need a 2:2 bachelor’s degree with honours or higher.
-
-[Contact accredited early years training providers to apply for a place](https://www.gov.uk/government/publications/early-years-initial-teacher-training-itt-providers-and-school-direct-early-years-lead-organisations/early-years-initial-teacher-training-itt-providers-and-school-direct-lead-organisations).
+### Non-UK candidates
```

Comment:
Thanks for adding this in @philspray !

Rather than having it as an h3, should we be using our inset component - wasn't this the way we were going to flag different info for non-uk candidates across the site? Paging @EllieNodder as I might be wrong!

Reply:
yeah I think the inset component here makes sense